conda_requirements.yml

```diff
@@ -1,56 +1,56 @@
-name: nwtenv
+name: NWTOPT
 channels:
-- conda-forge
-- defaults
+  - conda-forge
+  - defaults
 dependencies:
-- _libgcc_mutex=0.1
-- _openmp_mutex=4.5
-- blas=1.0
-- bottleneck=1.3.2
-- ca-certificates=2021.7.5
-- certifi=2021.5.30
-- cloudpickle=1.6.0
-- colorama=0.4.4
-- conda-pack
-- decorator=5.0.9
-- future=0.18.2
-- hyperopt=0.2.5
-- ld_impl_linux-64=2.35.1
-- libblas=3.9.0
-- libcblas=3.9.0
-- libffi=3.3
-- libgcc-ng=9.3.0
-- libgfortran-ng=11.1.0
-- libgfortran5=11.1.0
-- libgomp=9.3.0
-- liblapack=3.9.0
-- libopenblas=0.3.17
-- libstdcxx-ng=9.3.0
-- ncurses=6.2
-- networkx=2.3
-- numexpr=2.7.3
-- numpy=1.21.1
-- openssl=1.1.1k
-- pandas=1.3.0
-- pip=21.2.2
-- pymongo=3.12.0
-- python=3.8.11
-- python-dateutil=2.8.2
-- python_abi=3.8
-- pytz=2021.1
-- readline=8.1
-- scipy=1.7.0
-- setuptools=52.0.0
-- six=1.16.0
-- sqlite=3.36.0
-- tk=8.6.10
-- tqdm=4.62.0
-- wheel=0.36.2
-- xz=5.2.5
-- zlib=1.2.11
-- pip
-- pathlib=1.0
-- dill=0.3.4
-- wrapt=1.12.1
-- multiprocess=0.70.12.2
-
+  - _libgcc_mutex=0.1=main
+  - _openmp_mutex=4.5=1_gnu
+  - blas=1.0=openblas
+  - bottleneck=1.3.2=py38heb32a55_1
+  - ca-certificates=2021.7.5=h06a4308_1
+  - certifi=2021.5.30=py38h06a4308_0
+  - cloudpickle=1.6.0=py_0
+  - colorama=0.4.4=pyh9f0ad1d_0
+  - decorator=5.0.9=pyhd8ed1ab_0
+  - future=0.18.2=py38h578d9bd_3
+  - hyperopt=0.2.5=pyh9f0ad1d_0
+  - ld_impl_linux-64=2.35.1=h7274673_9
+  - libblas=3.9.0=10_openblas
+  - libcblas=3.9.0=10_openblas
+  - libffi=3.3=he6710b0_2
+  - libgcc-ng=9.3.0=h5101ec6_17
+  - libgfortran-ng=11.1.0=h69a702a_6
+  - libgfortran5=11.1.0=h6c583b3_6
+  - libgomp=9.3.0=h5101ec6_17
+  - liblapack=3.9.0=10_openblas
+  - libopenblas=0.3.17=pthreads_h8fe5266_1
+  - libstdcxx-ng=9.3.0=hd4cf53a_17
+  - ncurses=6.2=he6710b0_1
+  - networkx=2.3=py_0
+  - numexpr=2.7.3=py38h4be448d_1
+  - numpy=1.21.1=py38h9894fe3_0
+  - openssl=1.1.1k=h27cfd23_0
+  - pandas=1.3.0=py38h295c915_0
+  - pip=21.2.2=py38h06a4308_0
+  - pymongo=3.12.0=py38h709712a_0
+  - python=3.8.11=h12debd9_0_cpython
+  - python-dateutil=2.8.2=pyhd3eb1b0_0
+  - python_abi=3.8=2_cp38
+  - pytz=2021.1=pyhd3eb1b0_0
+  - readline=8.1=h27cfd23_0
+  - scipy=1.7.0=py38h7b17777_1
+  - setuptools=52.0.0=py38h06a4308_0
+  - six=1.16.0=pyh6c4a22f_0
+  - sqlite=3.36.0=hc218d9a_0
+  - tk=8.6.10=hbc83047_0
+  - tqdm=4.62.0=pyhd8ed1ab_0
+  - wheel=0.36.2=pyhd3eb1b0_0
+  - xz=5.2.5=h7b6447c_0
+  - zlib=1.2.11=h7b6447c_3
+  - pip:
+    - dill==0.3.4
+    - multiprocess==0.70.12.2
+    - pathlib==1.0.1
+    - wrapt==1.12.1
+    - wrapt-timeout-decorator==1.3.1
+prefix: /home/mnewcomer/anaconda3/envs/NWTOPT
```

Comment:
also if someone tries to build this env I think they will have a problem because of this path. related to specificity noted above

Reply:
Sounds good, I'll fix